### ISSUE TITLE
Without .remove([]), how do we exclude built-in utils?

### ISSUE THREAD

**Issue by davidmerrique** [jongacnik/gr8#18]
Seems like with 3.0.0 it's not very easy to exclude any of the default utils anymore unless doing something using the proxies - and then breakpoints aren't generated.

I'd like to use just a few of the defaults and some custom utils with the breakpoints feature.

Perhaps there could be an exclude option?

```javascript
const css = gr8({
  breakpoints: {
    sm: 768,
    md: 1024,
    lg: 1280
  },
  exclude: ['column', 'margin']
  utils: [customUtil]
});
```

Thanks!

**Comment by jongacnik** [comment]
Yo! Yes, you're right about that. I guess I incorrectly assumed that it wasn't getting much use. I think bringing in exclude as an option to pass in util keys will work as a patch (can likely do this today/tomorrow).

I might put a little bit of thinking into how this can ultimately become even more modular, given your use-case.

Rad ✨

**Comment by jongacnik** [comment]
@davidmerrique Let me know if #19 assists with your use-case

**Comment by davidmerrique** [comment]
@jongacnik That's perfect! 

Not sure if you want to go further and make it like before where you could be more specific and exclude things like `column.offset`, `column.nestedColumn`, `column.nestedOffset`, etc. 

This is great, thank you

**Comment by jongacnik** [comment]
@davidmerrique do you find you used that level of specificity? nested columns and nested offsets are gone btw! helps reduce the bloat a ton.

**Comment by davidmerrique** [comment]
@jongacnik I did use it a bit, but it's not a huge priority. The basic exclude option is just right.

**Comment by jongacnik** [comment]
Cool, gonna keep just as top level excludes for now. Merged! #19
